feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/AptoideAppsRepository.kt

```diff
@@ -324,23 +314,28 @@ fun AppJSON.toDomainModel(
   aab = mapAab(this),
   obb = mapObb(this),
   developerName = this.developer?.name,
-  campaigns = this.urls.mapCampaigns(campaignRepository, campaignUrlNormalizer)
+  campaigns = this.urls.mapCampaigns(campaignRepository)
     ?.apply { this.adListId = adListId }
 )
 
 fun CampaignUrls.mapCampaigns(
   campaignRepository: CampaignRepository?,
-  campaignUrlNormalizer: CampaignUrlNormalizer?,
 ): CampaignImpl? {
-  if (campaignRepository != null && campaignUrlNormalizer != null) {
-    val impressionsList = this.impression?.map { campaignUrl -> campaignUrl.url } ?: emptyList()
-    val clicksList = this.click?.map { campaignUrl -> campaignUrl.url } ?: emptyList()
+  if (campaignRepository != null) {
+    val impressionsList =
+      this.impression?.map { campaignUrl -> CampaignTuple(campaignUrl.name, campaignUrl.url) }
```

Comment:
There could be an extension function on List<CampaignUrl>? that maps the items to CampaignTuple, since its reused